developer-docs/plugin-guide.md

```diff
@@ -236,42 +184,36 @@ from pyglui import ui
 
 
 class Custom_Plugin(Plugin):
+    # Calling add_menu() will create an icon in the icon bar that represents
+    # your plugin. You can customize this icon with a symbol of your choice.
+    icon_chr = '@'  # custom menu icon symbol
+
+    # The default icon font is Roboto: https://fonts.google.com/specimen/Roboto
+    # Alternatively, you can use letters from the Pupil Icon font:
```

Comment:
Alternatively, you can use **icons** from the Pupil Icon font

(not letters, icons, or symbols if you like)